### PR TITLE
ARS-900: confidentiality status on view application pdf

### PR DESCRIPTION
**Pull request by C-Thomp**
image

### CODE REVIEW

File: app/views/components/AboutTheGoods.scala.html [hunk 1]
```diff
@@ -17,35 +17,40 @@
...
-    heading: Heading,
-    headingH3: headingH3,
-    paragraph: Paragraph
+layout: templates.Layout,
+govukSummaryList: GovukSummaryList,
+applicationSummaryList: ApplicationSummaryList,
+heading: Heading,
+headingH3: headingH3,
+paragraph: Paragraph,
+govukTable: GovukTable,
```
Comment (RawToast) [inline app/views/components/AboutTheGoods.scala.html:30]: I don't think it's used here either

File: app/views/components/ApplicationSummaryList.scala.html [hunk 2]
```diff
@@ -14,22 +14,6 @@
  * limitations under the License.
  *@
 
-@*
```
Comment (RawToast) [inline app/views/components/ApplicationSummaryList.scala.html:17]: 🧹

File: app/views/components/AboutTheGoods.scala.html [hunk 3]
```diff
@@ -17,35 +17,40 @@
...
 )
 
-@(appModel: ApplicationViewModel, referenceNumber: String, lastUpdated: String)(implicit messages: Messages)
+@(appModel: ApplicationViewModel, referenceNumber: String, lastUpdated: String, attachments: Seq[Attachment])(implicit messages: Messages)
```
Comment (RawToast) [inline app/views/components/AboutTheGoods.scala.html:33]: Perhaps it's just the diff, but I believe attachments is not used here 🤔

File: app/viewmodels/application/GoodsDetailsSummary.scala [hunk 4]
```diff
@@ -53,19 +58,29 @@ object GoodsDetailsSummary {
...
+    val confidentialInformationRow: Option[SummaryListRow] =
+      goodsDetails.confidentialInformation.map {
+        info =>
+          SummaryListRowViewModel(
+            key = "confidentialInformation.checkYourAnswersLabel",
+            value = ValueViewModel(info)
+          )
+      }
+
+    val attachmentRowContent = attachments
```
Comment (RawToast) [inline app/viewmodels/application/GoodsDetailsSummary.scala:70]: I believe this could be covered with an additional case in GoodsDetailsSummarySpec

**Review by RawToast — COMMENTED** [review]
I just released this was a draft PR 🤯

File: app/views/ViewApplicationView.scala.html [hunk 5]
```diff
@@ -32,10 +35,11 @@
         forYourRecords: ForYourRecords,
         appConfig: FrontendAppConfig,
         fullMainContent: FullLayoutMainContent,
-        printApplicationLink: PrintApplicationLink
+        printApplicationLink: PrintApplicationLink,
+        govukTable: GovukTable
```
[the remark this answers is not in the document]
Comment (C-Thomp) [reply app/views/ViewApplicationView.scala.html:39]: Not anymore, i had an iteration of this that used a table and looked terrible 🧹

File: app/views/components/AboutTheGoods.scala.html (hunk 1 above)
Comment (C-Thomp) [reply app/views/components/AboutTheGoods.scala.html:30]: Not anymore, i had an iteration of this that used a table and looked terrible 🧹

File: app/viewmodels/application/GoodsDetailsSummary.scala (hunk 4 above)
Comment (C-Thomp) [reply app/viewmodels/application/GoodsDetailsSummary.scala:70]: Added

**Review by RawToast — DISMISSED** [review]
🎉

File: app/views/ViewApplicationView.scala.html [hunk 6]
```diff
@@ -35,8 +35,7 @@
         forYourRecords: ForYourRecords,
         appConfig: FrontendAppConfig,
         fullMainContent: FullLayoutMainContent,
-        printApplicationLink: PrintApplicationLink,
-        govukTable: GovukTable
+        printApplicationLink: PrintApplicationLink
 )
 
 @(appModel: ApplicationViewModel, applicationId: String, lastUpdated: String, attachments: Seq[Attachment])(implicit request: Request[_], messages:
```
[the remark this answers is not in the document]
Comment (C-Thomp) [reply app/views/ViewApplicationView.scala.html]: true, tidied up :)

**Pull request merged**